packages/bpk-foundations-common/base/colors/button.json

```diff
@@ -95,11 +71,17 @@
       "PRIVATE_BUTTON_PRIMARY_ON_LIGHT_PRESSED_BACKGROUND_NIGHT": {
         "value": "#154679"
       },
-      "PRIVATE_BUTTON_PRIMARY_ON_LIGHT_LOADING_BACKGROUND_DAY": {
-        "value": "#154679"
+      "PRIVATE_BUTTON_PRIMARY_ON_DISABLED_BACKGROUND_DAY": {
```

Comment:
naming nitpick: `disabled` rather than `on_disabled`?
also wonder if we should have a default disabled background & foreground and then just have the few special cases as specific tokens?

Reply:
ah i see now - I would maybe just have two separate tokens for these so there's no confusion about when which ones are used where - ie `privateButtonPrimaryOnLightDisabledBackground` etc
OR: `privateButtonDisabledDarkBackground` (so basically have a dark/light version of it which then refers to the dark/light one in day + night mode